src/massive/munit/TestRunner.hx

```diff
@@ -189,7 +189,11 @@ class TestRunner implements IAsyncDelegateObserver
 
         #if (neko || cpp || java || hl || eval)
 		var self = this;
-		var runThread:Thread = Thread.create(function()
+		var createThread = Thread.create;
+		#if (haxe4 && (haxe >= version("4.2.0-rc.1")))
```

Comment:
why not `#if (haxe >= version("4.2.0-rc.1"))`?

Reply:
mb?
```haxe
var createThread = #if (haxe >= version("4.2.0-rc.1")) Thread.createWithEventLoop #else Thread.create #end;
```

Reply:
I didn't want to break compatibility with Haxe 3, `version("x.x.x")` is a Haxe 4 feature

Reply:
that being said... I should have tested with Haxe 3, which makes a bit more ugly. I had to split the conditionals to make it work.